src/core_tests.rs

```diff
@@ -14,6 +14,7 @@
 //
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
+#![cfg(test)]
```

Comment:
We have already have a `cfg` for this module in [lib.rs](https://github.com/maidsafe/routing/blob/master/src/lib.rs#L175) - I guess we should merge these?